src/sass/components/ATATPhoneInput.scss

```diff
@@ -5,7 +5,7 @@
     .v-input__control > {
 
       .v-input__slot {
-        padding-top: 8px;
+        padding-top: 22px;
```

Comment:
this is getting overwritten by another class. is this needed?

Reply:
this was used to center the text in the dropdown search bar but it will be handled in the phone input tech debt ticket